test/unit/auth.js

```diff
@@ -7,6 +7,8 @@ import {oauthFlow, AccessToken} from '../../src/auth_v3'
 import mock from '../mock-api'
 import {decodeQuery} from '../../src/utils'
 
+global.navigator = undefined
```

Comment:
I don't know why but test failed if I remove this line ;) https://travis-ci.org/cozy/cozy-client-js/builds/222804829#L640